packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -75,13 +75,16 @@ export interface IContainerRuntimeMetadata {
     readonly summaryFormatVersion: 1;
     /** True if channels are not isolated in .channels subtrees, otherwise isolated. */
     readonly disableIsolatedChannels?: true;
+    /** True to enable GC, false to disable GC, undefined to not specify a preference. */
+    readonly enableGC?: boolean;
```

Comment:
i don't like that enableGc means on thing here, and somethings else on the container, please unify the config names

Reply:
I don't know how much it would help, but maybe I can change to `summaryAllowGC` ?
`summary` - to indicate it is per document / stored in summary
`allow` - vs. enable to show that this is less of an option more of a restriction?

@agarwal-navin also thoughts?

Reply:
I am wondering if having both `enableGC / summaryEnableGC` and `disableGC` is a bit too much. It is confusing for sure. We can probably just have `enableGC` and the override (to disable it) can be done via the local storage or via loader options (which would be hidden).